configure.py

```diff
@@ -380,12 +380,12 @@ def process_command_line(args):
         'AltiVec', 'NEON', 'ARMv8 Crypto', 'POWER Crypto']
 
     for isa_extn_name in isa_extensions:
-        isa_extn = isa_extn_name.lower().replace(' ', '')
+        isa_extn = isa_extn_name.lower().replace(' ', '').replace('.', '')
 
         target_group.add_option('--disable-%s' % (isa_extn),
                                 help='disable %s intrinsics' % (isa_extn_name),
                                 action='append_const',
-                                const=isa_extn.replace('-', '').replace('.', '').replace(' ', ''),
+                                const=isa_extn.replace('-', '').replace(' ', ''),
```

Comment:
For the sake of backward compatibility (e.g. with the Conan recipe), maybe add the switches that don't have a period (`.`) instead of replacing them with the existing ones.

https://github.com/conan-io/conan-center-index/blob/f89d800e5d4eb8d42af1560bf2cc28ac894428f6/recipes/botan/all/conanfile.py#L329-L333

Reply:
Good point, will take care.

Reply:
I reverted this change, it is not worth changing this now. Maybe make it POSIX-compliant in 4.0. I formatted the options manually instead.